docs/dev/framework/cron.md

```diff
@@ -64,6 +64,11 @@ a Linux crontab you could use the following instructions for example:
 * * * * * php /path/to/contao/vendor/bin/contao-console contao:cron
 ```
 
+{{% notice tip %}}
+On the command line is no HTTP request context available. Contao needs this for example 
+to generate the sitemap. You can set the domain in the root site configuration or via 
+configuration file. See [Symfony Routing Documentation](https://symfony.com/doc/current/routing.html#generating-urls-in-commands)
```

Comment:
```suggestion
There is no HTTP request context available on the command line. However, Contao needs
this to generate the sitemap for example. You can set the domain either in the settings of
your website roots or you can define a default domain in your application configuration.
See the [Symfony Routing Documentation](https://symfony.com/doc/4.4/routing.html#generating-urls-in-commands) for more details.
```

I have changed the link to the 4.4 documentation, since the 5.1 (current) one doesn't actually explain the relevant parameters unfortunately. Can you also add the example code like this? Couldn't do it via suggestion.

```yaml
# config/parameters.yml
parameters:
    router.request_context.host: 'example.org'
    router.request_context.scheme: 'https'
```